plugins/dos2unix/README.md

```diff
@@ -0,0 +1,9 @@
+## Dos2unix filter
```

Comment:
Sorry I forgot about this part, I should have been clearer in my previous review comment. Could you break out the dos2unix-part to its own commit? That way we get a clean _mechanisms_-commit and then a separate commit for each new feature.